### PR TITLE
Work with new registration method, blade templating and add unpublish check

### PR DESCRIPTION
**Pull request by michaelclaesson**

### CODE REVIEW

File: source/php/Module/views/noticeboard.blade.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        @forelse ($postItems as $postItem)
+            <li>
+                @if ($postItem['link'])
+                    <a href="{{ $postItem['link'] }}">{{ $postItem['title'] }} {{ $postItem['meetingDate'] }}</a>
+                @else
+                    {{ $postItem['title'] }} {{ $postItem['meetingDate'] }}
+                @endif
+                Anslaget: {{ $postItem['published'] }}
+                @if ($postItem['unpubDate'])
+                    Tas ner: {{ $postItem['unpubDate'] }}
```
Comment (sebastianthulin) [inline source/php/Module/views/noticeboard.blade.php]: Static strings should be handled with language functions __() and be written in english. These needs to reside outside .blade.php files to be indexable with poedit.

File: source/php/Module/views/noticeboard.blade.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                @endif
+            </li>
+        @empty
+            <li>Inga inlägg tillgängliga.</li>
```
Comment (sebastianthulin) [inline source/php/Module/views/noticeboard.blade.php]: Static strings should be handled with language functions __() and be written in english. These needs to reside outside .blade.php files to be indexable with poedit.

File: source/php/Module/views/noticeboard.blade.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    <ul>
+        @forelse ($postItems as $postItem)
+            <li>
+                @if ($postItem['link'])
+                    <a href="{{ $postItem['link'] }}">{{ $postItem['title'] }} {{ $postItem['meetingDate'] }}</a>
+                @else
+                    {{ $postItem['title'] }} {{ $postItem['meetingDate'] }}
+                @endif
+                Anslaget: {{ $postItem['published'] }}
```
Comment (sebastianthulin) [inline source/php/Module/views/noticeboard.blade.php]: Static strings should be handled with language functions __() and be written in english. These needs to reside outside .blade.php files to be indexable with poedit.

File: source/php/Module/views/noticeboard.blade.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<div class="{{ $classes }}">
```
Comment (sebastianthulin) [inline source/php/Module/views/noticeboard.blade.php:1]: This view would benefit from utlizing the component library.

File: source/php/Module/views/noticeboard.blade.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    <h4 class="box-title">{{ $postTitle }}</h4>
+
+    @if (!empty($introductoryText))
+        {!! wpautop($introductoryText) !!}
```
Comment (sebastianthulin) [inline source/php/Module/views/noticeboard.blade.php]: Functions inside of blade templates should be avoided.

**Review by sebastianthulin — APPROVED** [review]
Left some comments, but no requirement to make these changes to merge. Let us know what you want to do!

**Comment by michaelclaesson** [comment]
@sebastianthulin I've pushed a few changes. Hopefully this is what you're looking for! ☺️

**Review by sebastianthulin — APPROVED** [review]

**Pull request merged**